pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +143,72 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+// Plugins include enabled plugins and disabled plugins, when enabled, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
+type Plugins struct {
+	// EnabledPlugins specifies plugins that should be enabled in addition to default plugins.
+	EnabledPlugins *PluginSet
+	// DisabledPlugins specifies default plugins that should be disabled.
```

Comment:
need to clearly describe how to disable all default plugins. enumerate all default plugins in the disabled list (if so, how do they guard against new default plugins on upgrade)? include one with `name: *` in the disabled list?

Reply:
Another great point. We should support "disable all", although disabling plugins is not implemented yet. We still should have a plan for it and document it in the API.